Vagrantfile

```diff
@@ -12,7 +12,7 @@ end
 Vagrant.configure("2") do |config|
     config.vm.box = "bento/ubuntu-18.04"
     config.vm.network "forwarded_port", guest: 80, host: 81, host_ip: "127.0.0.1"
-    config.vm.network "forwarded_port", guest: 8000, host: 8082, host_ip: "127.0.0.1"
+    config.vm.network "forwarded_port", guest: 8000, host: 8084, host_ip: "127.0.0.1"
```

Comment:
Not relevant for this project, but when starting a new project, we refer to the [team repo](https://github.com/azavea/geospatial-apps) to make sure that none of the new ports we assign conflict with existing projects.